.github/workflows/e2e.yml

```diff
@@ -12,19 +12,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        cable_driver: ['libreswan', 'wireguard']
         deploytool: ['operator']
         globalnet: ['', 'globalnet']
-        cable_driver: ['libreswan', 'wireguard']
-        ovn: ['', 'ovn']
         k8s_version: ['1.20.2']
-        exclude:
-          - ovn: 'ovn'
-            globalnet: 'globalnet'
-          - ovn: 'ovn'
-            cable_driver: 'wireguard'
+        ovn: ['']
         include:
           # This is the oldest K8s version we try to support
           - k8s_version: 1.17.17
+          - ovn: 'ovn'
```

Comment:
Should we be more verbose here and list/duplicate all the defaults so they show in the PR UI? It might be simpler for folks who don't know the defaults to compare what components are tested in which jobs.